### PR TITLE
Fix panic when a command fails + more

### LINKED ISSUE #9: Check if lscpu/lspci/lsusb commands exist
Some distros might not ship one of these commands ootb (e.g. Alpine Linux/postmarketOS), and not having one of them available will cause the app to panic, it should be simple enough to add a check to see if the output is an `Err` or not.

### PR DESCRIPTION
**Pull request by sungsphinx**
Apologies for most of this being in one whole commit, I had already done quite a lot of code changes and it would be a pain to split it.

Main changes:
- Make wgpu an optional feature (useful for operating systems like Redox OS)
- Fix panicking when either lspci, lspcu or lsusb fail
- If a command fails, it will try to show the error message on the respective page
- Add usbutils to Flatpak, which should make the lsusb command available
- Add cosmic-icons to Flatpak

Minor-ish changes:
- Make the log/output in the terminal prettier
- Change some code to pass clippy
- Change "Uh oh, something went wrong" to "An error occurred"
- Make the app icon bigger

Fixes #9

### CODE REVIEW

File: page.codeberg.sungsphinx.Examine.json [hunk 1]
```diff
@@ -59,6 +59,71 @@
...
+		{
+		    "type": "archive",
+		    "url": "https://github.com/libusb/libusb/releases/download/v1.0.27/libusb-1.0.27.tar.bz2",
+		    "sha256": "ffaa41d741a8a3bee244ac8e54a72ea05bf2879663c098c82fc5757853441575"
+		}
+	    ],
+	    "post-install": [
+		"install -Dm644 COPYING ${FLATPAK_DEST}/share/licenses/libusb/COPYING"
+	    ]
+	},
```
Comment (edfloreshz) [inline page.codeberg.sungsphinx.Examine.json]: This entry has some formatting issues.

**Review by edfloreshz — CHANGES_REQUESTED** [review]
Just a minor change

File: page.codeberg.sungsphinx.Examine.json (hunk 1 above)
Comment (sungsphinx) [reply page.codeberg.sungsphinx.Examine.json]: Should be fixed, unless that's not what you meant :sweat_smile:

**Review by edfloreshz — APPROVED** [review]
LGTM!

**Comment by sungsphinx** [comment]
Should I switch to the cosmic baseapp before merging @edfloreshz?

**Comment by edfloreshz** [comment]
I still haven't gotten it to build, so I wouldn't recommend it just yet.

**Pull request merged**